dts/bindings/video/st,stm32-venc.yaml

```
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2025 STMicroelectronics.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    STMicroelectronics STM32 video encoder peripheral (VENC).
+    Example of node configuration at board level:
```

Stray line remaining
```suggestion
description: |
    STMicroelectronics STM32 video encoder peripheral (VENC).
```

But can simplify to:
```suggestion
description: STM32 video encoder (VENC)
```